01. 기본문제 풀이/01. 세 수 중 최솟값/김보경/index.js

```diff
@@ -1,7 +1,29 @@
 function solution(a, b, c) {
   let answer;
+  if (a < b) {
+    answer = a;
+  } else {
+    answer = b;
+  }
+
+  if (answer < c) {
+    return;
+  } else {
+    answer = c;
+  }
```

Comment:
c가 answer보다 작은 경우만 비교하고 else문은 없어도 되지 않을까요?

그냥 return만 하면 세 수 중 최솟값이 아닌 undefined가 반환될 거 같아요.
한다면 `return answer;`로 해야할 거 같아요
```suggestion
  if (c < answer) {
    answer = C;
  }
```